e2e-tests/users/run

```diff
@@ -50,11 +54,32 @@ sleep 15
 wait_cluster_consistency "$cluster" 3 1
 compare_mysql_cmd "select-4" "SHOW TABLES;" "-h $cluster-proxysql -uoperator -p$newpass"
 
+desc 'change secret name'
+apply_config "$test_dir/conf/some-name.yml"
```

Comment:
can we change the secrets name via `kubectl_bin patch pxc some-name ...` command.
reason: for better visibility what is changed

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/495/commits/6171bb243e5413236f6443a988a4b7065bad47ea